bin/morgue.js

```diff
@@ -7437,6 +7438,136 @@ async function metricsImporterCmd(argv, config) {
   await cli.routeMethod(argv);
 }
 
+function stabilityCreateMetric(coroner, argv, config) {
+  let universe = argv.universe;
+  if (!universe) {
+    universe = Object.keys(config.config.universes)[0];
+  }
+
+  let project = argv.project;
+
+  if (!universe) {
+    errx("--universe is required");
+  }
+
+  if (!project) {
+    errx("--project is required");
+  }
+
+  let bpg = coronerBpgSetup(coroner, argv);
+  let model = bpg.get();
+
+  /* Find universe. */
+  let uid = 0;
+  for (let u of model.universe) {
+    if (u.get("name") === universe) {
+      uid = u.get("id");
+    }
+  }
+
+  if (uid == 0) {
+    errx("Universe not found");
+  }
+
+  let pid = 0;
+  for (let p of model.project) {
+    if (p.get("universe") === uid && p.get("name") == project) {
+      pid = p.get("pid");
+    }
+  }
+
+  if (pid === 0) {
+    errx("Project not found");
+  }
+
+  const metricGroupName = argv["metric-group"];
+  if (!metricGroupName) {
+    errx("--metric-group is required");
+  }
+  if (Array.isArray(metricGroupName)) {
+    errx("SPecify only one --metric-group");
+  }
+
+  const name = argv.name;
+  if (!name) {
+    errx("--name is required");
+  }
+  if (Array.isArray(name)) {
+    errx("Only one --name allowed");
+  }
+
+  let metricGroupId = 0;
+
+  if (!model.metric_group) {
+    errx("No metric groups exist yet.");
+  }
+
+  for (let g of model.metric_group) {
+    if (g.get("name") === metricGroupName &&
+      g.get("project") === pid) {
+        metricGroupId = g.get("id");
+    }
+  }
+
+  if (metricGroupId === 0) {
+    errx("Metric group not found");
+  }
+
+  let attributes = argv.attribute;
+  if (attributes === undefined) {
+    /*
+     * 0 attributes are possible, though rare save for people integrating with
+     * the submission API directly.
+     */
+    attributes = [];
+  } else if (!Array.isArray(attributes)) {
+    attributes = [ attributes ];
+  }
+
+  let attributesObj = {};
+  for (let a of attributes) {
+    const parts = a.split(",");
+    if (parts.length != 2) {
+      errx("Usage of --attribute is name, value");
+    }
+    /*
+     * This works because coronerd uses crdb_column_string_set, so no matter
+     * the column type, the attribute can be a string. Otherwise, we'd have
+     * to do a bunch of joins in order to do further validation.
+     */
+    attributesObj[parts[0]] = parts[1];
+  }
+
+  const obj = bpg.new("metric").withFields({
+    metric_group: metricGroupId,
+    name,
+    attribute_values: JSON.stringify(attributesObj),
+  });
+  bpg.create(obj);
+  bpg.commit();
+  console.log("Metric created");
+}
+
+function coronerStability(argv, config) {
+  const coroner = coronerClientArgv(config, argv);
+
+  argv._.shift();
+  if (argv._.length === 0) {
```

Comment:
Do we know about other kinds of metrics integration under the `stability` namespace that we will use In the future?

Reply:
I'm not sure what you're asking here.  The stability namespace isn't a metrics namespace, it's allowing someone to interact with a specific feature for stability.  Technically it could be used by a sufficiently creative user for other sorts of metrics, I suppose, but the only reasonable use is for stability and we need `metrics-importer` for the service (plus you can get metrics in here without going through metrics-importer, as well).

Reply:
My point is - right now you have only one command "create-metrics" - do you plan to add more commands?